canonical-web-and-design/maas.io
# remove jQuery scripts

## Done
- **remove jquery from the site entirely**
- move venobox modal script to the tour page, reducing size for all other pages

### jquery use in maas.io
#### scripts in head.html:
- `<script src="/static/js/jquery.modal.min.js" defer></script>`
jquery venobox modal, refactored and **replaced with vanilla js version**

- `<script src="/static/js/jquery.command-line.js" defer></script>`
 **removed**
does not appear to be used (queries the DOM for `.p-code-copyable` which does not appear anywhere in maas.io)

- `<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>`
**removed**


## QA

- Go to the `/tour` page and
- ensure all images still open the modal correctly (you can compare with the current version - https://maas.io/tour )

## Issue / Card

#653 

## Screenshots

### Before
![image](https://user-images.githubusercontent.com/7452681/154229973-be36a5a3-6275-4ab1-a558-d30f4c81d6f9.png)

### After
![image](https://user-images.githubusercontent.com/7452681/154229911-312bfa68-d7c8-4b20-9c3a-089f7ea128bc.png)
